py3status/modules/xrandr_toggle.py

```diff
@@ -88,7 +88,7 @@ def _xrandr(self):
                         monitors.append(source)
             return monitors
         except:
-            return self._error('error while parsing xrandr', None)
+            return self._error('error while parsing xrandr')
```

Comment:
I was thinking of actually setting the return value to None, and in `get_status()` I would first check if monitors is set to None to notice the user of a xrandr parsing error.

Wouldn't that make it easier to catch an error from there?